### PR TITLE
Feature/screens

### PR DESCRIPTION
**Pull request by hdrabs**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,2 @@
+{
```
Comment (karamat) [inline .vscode/settings.json]: add it .ignore

File: App/Components/CountryDropdown/index.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
...onia', 'Madagascar', 'Malawi', 'Malaysia', 'Maldives', 'Mali', 'Malta', 'Mauritania', 'Mauritius', 'Mexico', 'Moldova', 'Monaco', 'Mongolia', 'Montenegro', 'Montserrat', 'Morocco', 'Mozambique', 'Namibia', 'Nepal', 'Netherlands', 'Netherlands Antilles', 'New Caledonia', 'New Zealand', 'Nicaragua', 'Niger', 'Nigeria', 'Norway', 'Oman', 'Pakistan', 'Palestine', 'Panama', 'Papua New Guinea', 'Paraguay', 'Peru', 'Philippines', 'Poland', 'Portugal', 'Puerto Rico', 'Qatar', 'Reunion', 'Romania', 'Russia', 'Rwanda', 'Saint Pierre &amp; Miquelon', 'Samoa', 'San Marino', 'Satellite', 'Saudi Arabia', 'Senegal', 'Serbia', 'Seychelles', 'Sierra Leone', 'Singapore', 'Slovakia', 'Slovenia', 'South Africa', 'South Korea', 'Spain', 'Sri Lanka', 'St Kitts &amp; Nevis', 'St Lucia', 'St Vincent', 'St. Lucia', 'Sudan', 'Suriname', 'Swaziland', 'Sweden', 'Switzerland', 'Syria', 'Taiwan', 'Tajikistan', 'Tanzania', 'Thailand', "Timor L'Este", 'Togo', 'Tonga', 'Trinidad &amp; Tobago', 'Tunisia', 'Turkey', 'Turkmenistan', 'Turks &amp; Caicos', 'Uganda', 'Ukraine', 'United Arab Emirates', 'United Kingdom', 'Uruguay', 'Uzbekistan', 'Venezuela', 'Vietnam', 'Virgin Islands (US)', 'Yemen', 'Zambia', 'Zimbabwe']
```
Comment (karamat) [inline App/Components/CountryDropdown/index.js]: move it to state

File: App/Components/Paperwork/index.js [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+*
+* Paperwork
+*
+* Creates a component to add paperwork from the camera roll OR opens camera to take a picture.
+*
+*/
+
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { TouchableOpacity, Dimensions, TouchableHighlight, Image, Text, CameraRoll, View, Modal, Button, ScrollView } from 'react-native'
```
Comment (karamat) [inline App/Components/Paperwork/index.js]: break in 2 lines

File: App/Components/Paperwork/index.js [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+  setIndex = (index) => {
+    if (index === this.state.index) {
+      index = null
+    }
+    this.setState({ index })
+    console.log(index)
+    index !== null
+    ? this.imagePath = this.state.photos[index].node.image.uri : this.imagePath = ''
+    console.log(this.imagePath) // CONSOLING THE URI OF SELECTED IMAGE
```
Comment (karamat) [inline App/Components/Paperwork/index.js]: better remove these

File: App/Components/Paperwork/index.js [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+        >
+          <View style={styles.modalContainer}>
+            { this.state.index === null
+            ? <Button
+              title='Close'
+              onPress={() => { this.toggleModal(); this.props.captureImageUri(null) }}
+            />
+            : <Button
+              title='Done'
+              onPress={() => { this.toggleModal(); this.props.captureImageUri(this.state.photos[this.state.index].node.image.uri) }}
```
Comment (karamat) [inline App/Components/Paperwork/index.js]: break into multiples lines

File: App/Containers/LaunchScreen/index.js [hunk 6]
```diff
@@ -1,19 +1,28 @@
...
   static navigationOptions = ({ navigation }) => ({
     // title: `Chat with Me`,
   });
+
+  componentDidMount () {
+    setTimeout(() => {
+      this.props.navigation.navigate('SearchScreen')
+    }, 3000)
+  }
+  componentWillUnmount () {
```
Comment (karamat) [inline App/Containers/LaunchScreen/index.js]: remove

File: App/Containers/LaunchScreen/index.js [hunk 7]
```diff
@@ -1,19 +1,28 @@
...
+    console.log('componentWillUnmount')
+  }
+
   render () {
-    console.log(100);
+    console.log(this.props)
```
Comment (karamat) [inline App/Containers/LaunchScreen/index.js]: remove

File: App/Containers/MembershipInfo/index.js [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+        <Section>
+          <SwitchComponent />
+          <HeadingAndText text={'Membership Type'} />
+          <ButtonGroup
+            onPress={this.updateIndexType}
+            selectedIndex={this.state.typeIndex}
+            buttonStyle={{backgroundColor: 'white', borderWidth: 0}}
+            selectedButtonStyle={{borderWidth: 0}}
+            textStyle={{fontFamily: 'HelveticaNeueLTStd-Th', textAlign: 'left', fontSize: 18, color: Colors.searchFieldBackground}}
+            selectedTextStyle={{fontFamily: 'HelveticaNeueLTStd-Th', textAlign: 'right', fontSize: 18, color: Colors.darkGray}}
```
Comment (karamat) [inline App/Containers/MembershipInfo/index.js]: move these styles into `./styles`

File: App/Containers/PaymentInfo/index.js [hunk 9]
```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react'
+import { RedButton, CustomTextInput, Section, Container, Heading } from '../../Components'
+
+export default class PaymentInfo extends Component {
+  render () {
+    return (
+      <Container>
+        <Heading text={'Payment Info'} />
+        <Section>
+          <CustomTextInput label={'Name:'} />
```
Comment (karamat) [inline App/Containers/PaymentInfo/index.js]: rename CustomTextInput to simple TextInput

File: App/Components/CountryDropdown/index.js (hunk 2 above)
Comment (hdrabs) [reply App/Components/CountryDropdown/index.js]: Done

File: .vscode/settings.json (hunk 1 above)
Comment (hdrabs) [reply .vscode/settings.json]: Done

File: App/Components/Paperwork/index.js (hunk 3 above)
Comment (hdrabs) [reply App/Components/Paperwork/index.js]: Done

File: App/Components/Paperwork/index.js (hunk 4 above)
Comment (hdrabs) [reply App/Components/Paperwork/index.js]: Done

File: App/Components/Paperwork/index.js (hunk 5 above)
Comment (hdrabs) [reply App/Components/Paperwork/index.js]: Done

File: App/Containers/LaunchScreen/index.js (hunk 6 above)
Comment (hdrabs) [reply App/Containers/LaunchScreen/index.js]: Done

File: App/Containers/LaunchScreen/index.js (hunk 7 above)
Comment (hdrabs) [reply App/Containers/LaunchScreen/index.js]: Done

File: App/Containers/PaymentInfo/index.js (hunk 9 above)
Comment (hdrabs) [reply App/Containers/PaymentInfo/index.js]: Done

**Comment by hdrabs** [comment]
All the requested changes have been made to the pull request

**Pull request merged**